history.adoc

```diff
@@ -1,6 +1,6 @@
 :issues: https://github.com/cf-convention/cf-conventions/issues/
 :pull-requests: https://github.com/cf-convention/cf-conventions/pull/
-:tickets: https://cfconventions.org/Data/trac.html
+:tickets: https://cfconventions.org/Data/Trac-tickets/
```

Comment:
I might be missing some context, but the current link (with `trac.html`) works, and this suggestion doesn't, leading to a '404 - File not found' page when I check in the browser. Is there a corresponding page for Trac tickets that is due to be pushed from another branch to create this link, perhaps?

Reply:
Yes!, I have been very cryptic with this, it wasn't obvious to me neither. 

The link to the index of Trac tickets works:
> :tickets: https://cfconventions.org/Data/trac.html

but its purpose it's to define an attribute to be interpolated, as part of a complete link to the specific Trac ticket, later in the source file coded as, for example: 
> * {tickets}164.html[Ticket #164]: Add bounds attribute to first geometry CDL example.

Making all links to Trac tickets broken. Click them at::
https://cfconventions.org/Data/cf-conventions/cf-conventions-1.11/cf-conventions.html#_version_1_7_7_august_2017

Reply:
Aha, got it. Thanks for explaining, Antonio. That makes sense and I've checked some examples e.g. https://cfconventions.org/Data/Trac-tickets/164.html and they all work, so that's all good. Approving now.